src/users/users.service.ts

```diff
@@ -66,6 +68,48 @@ export class UsersService {
     return { user, token };
   }
 
+  // Get User Profile
+  async findOneById(id: string) {

```

Comment:
```suggestion
  async findById(id: string) {
```
dư này là đủ.
nếu cần define method mới để tìm nhiều bản ghi thì đặt tên là `findByIds`